community/teams/cuda.tt

```diff
@@ -0,0 +1,61 @@
+[% WRAPPER layout.tt title="CUDA Team" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html#governance-teams">Community</a>
+    <span>→</span>
+  </div>
+  <h1>CUDA team</h1>
+</div>
+
+<section class="governance-description">
+  The CUDA team is responsible for all things CUDA-related in <a href="https://nixpkgs.org">Nixpkgs</a>.
+</section>
+
+<section class="governance-team">
+  <aside>
+    <h2>Members</h2>
+    <ul>
+      <li>
+        Connor Baker (<a href="https://discourse.nixos.org/u/ConnorBaker">@ConnorBaker</a>)<br/>
+        CUDA maintainer, sponsored by <a href="https://tweag.io">Tweag</a>
+      </li>
+    </ul>
+
+    <h2>Contact</h2>
+    <ul>
+      <li><a href="https://discourse.nixos.org/t/announcing-the-nixos-cuda-maintainers-team-and-a-call-for-maintainers/18074">Discourse</a></li>
+      <li><a href="https://app.element.io/#/room/#cuda:nixos.org">Matrix</a></li>
+    </ul>
+  </aside>
+
+  <h2>Goals</h2>
+  <ul>
+    <li>
+      support community efforts involving packaging or distributing CUDA-enabled software
+    </li>
+    <li>
+      document CUDA packaging best practices, guidelines, and patterns
+    </li>
+    <li>
+      improve the quality of CUDA packaging
+    </li>
+    <li>
+      make Nix/NixOS the first choice for users of CUDA-related software, especially in machine learning and scientific computing
+    </li>
+  </ul>
+
+  <h2>Contact and Information</h2>
+  <p>
+    Discussions are held on Matrix, Discourse, and on GitHub issues and pull requests.
+  </p>
+  <p>
+    <a class="button" href="https://nixos.wiki/wiki/CUDA">Wiki</a>
```

Comment:
```suggestion
```

Probably shouldn't link to the Wiki, it's all over the place in terms of quality. This page in particular seems not very insightful. One thing this team could do for starters would be updating the [Nixpkgs manual section on CUDA](https://nixos.org/manual/nixpkgs/unstable/#cuda) a bit, and find ways to make the Wiki page obsolete (so we can eventually delete it), e.g. by adding guides and tutorials to nix.dev.

Reply:
Tbh the only reason I see the wiki page is still alive is that it links to cachix, which we don't include in the manual since there still isn't a reliable solution as to populating it, and no clear trust model. Replacing the wiki page with a clear cut tutorial on nix.dev sounds great

Reply:
Good points -- updated docs and tutorials would certainly be welcome and are something we need to work on.